mne/evoked.py

```diff
@@ -406,13 +406,17 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
             Color code lines by mapping physical sensor coordinates into color
             values. Spatially similar channels will have similar colors.
             Bad channels will be dotted.
+        legend : bool
+            Whether to show legend when using ``spatial_colors``. If
+            ``spatial_colors=False``, this parameter has no effect. Defaults
+            to True.
         """
         return plot_evoked(self, picks=picks, exclude=exclude, unit=unit,
                            show=show, ylim=ylim, proj=proj, xlim=xlim,
                            hline=hline, units=units, scalings=scalings,
                            titles=titles, axes=axes, gfp=gfp,
                            window_title=window_title,
-                           spatial_colors=spatial_colors)
+                           spatial_colors=spatial_colors, legend=legend)
```

Comment:
Yeah, that would be one option. With this you wouldn't be able to change the head/skirt topo like @jona-sassenhagen suggested though.

But in general, I think this would indeed be better.